src/tests/ftest/pool/rebuild_no_cap.py

```diff
@@ -48,8 +48,8 @@ def test_rebuild_no_capacity(self):
         :avocado: tags=all,medium,pr,pool,rebuild,nocap
         """
         # Get the test params
-        self.pool = TestPool(self.context, self.log,
-                             dmg_command=self.get_dmg_command())
+        self.pool = TestPool(self.context, self.get_dmg_command(),
+                             log=self.log)
```

Comment:
The `log` argument is defunct for TestPool.create():
```suggestion
        self.pool = TestPool(self.context, self.get_dmg_command())
```